vcsclient/bitbucketserver.go

```diff
@@ -701,3 +718,43 @@ func getBitbucketServerRepositoryVisibility(public bool) RepositoryVisibility {
 	}
 	return Private
 }
+
+// BitbucketParseCommitStatuses parse raw response into CommitStatusInfo slice
+// The response is the same for BitBucket cloud and server
+func BitbucketParseCommitStatuses(rawStatuses interface{}) ([]CommitStatusInfo, error) {
```

Comment:
Can be package-private
```suggestion
func bitbucketParseCommitStatuses(rawStatuses interface{}) ([]CommitStatusInfo, error) {
```